database/models.py

```diff
@@ -9,7 +10,22 @@ class Author(db.Document):
     description = db.StringField(required=True)
     createdOn = db.DateTimeField(default=datetime.datetime.utcnow)
     updatedOn = db.DateTimeField(default=datetime.datetime.utcnow)
-    meta = {'collection': 'authors'}
+    meta = {"collection": "authors"}
+
+    def to_json(self):
+        obj = self.to_mongo()
+        return objectToJson(obj)
+
+
+class ScrapedAuthor(db.Document):
+    name = db.StringField()
+    DOB = db.DateTimeField()
```

Comment:
```suggestion
    dob = db.DateTimeField()
```
Pick one of - date_of_birth, birth_date, dob